src/cargo/ops/cargo_package.rs

```diff
@@ -500,11 +513,7 @@ fn check_metadata(pkg: &Package, gctx: &GlobalContext) -> CargoResult<()> {
             )*
         }}
     }
-    lacking!(
-        description,
-        license || license_file,
-        documentation || homepage || repository
-    );
+    lacking!(description, documentation || homepage || repository);
```

Comment:
Why did `license || license_file` part get removed? It is still required to have license specified.

Reply:
I thought not specifying `license || license_file` was supposed to be an error and not a warning?

Reply:
Thanks. It has returned to its previous state.